File: curso-python/ex078.py
# leia 5 valores numéricos e guarde-os em uma lista
lista = []

for c in range(5):
    lista.append(int(input(f"Digite um valor para a posição {c}: ")))

# verificar os valores maiores e menores
maior = lista[0]
menor = lista[0]

# descobrir maior e menor
for valor in lista:
    if valor > maior:
        maior = valor
    if valor < menor:
        menor = valor

print("-="*50)
print(f"\nVocê digitou: {lista}")
print(f"o maior valor digitado foi {maior} nas posições ",end='')
for i,v in enumerate(lista):
    if v == maior:
        print(f"{i}... ",end=' ')

print()
print(f"o menor valor digitado foi {menor} nas posições ",end='')
for i,v in enumerate(lista):
    if v == menor:
        print(f"{i}... ", end=' ')

File: javascript-exercicios/ex078.js
const prompt = require("prompt-sync")();
let lista = [];

for (let c=0 ; c<5 ; c++){
    lista.push(Number(prompt(`digite um valor para a posição ${c}: `)));
}

let maior = Math.max(...lista);
let menor = Math.min(...lista);

console.log("-=".repeat(50));
console.log(`você digitou: ${lista}`);

console.log(`o maior valor digitado foi ${maior} nas posições`);
lista.forEach((valor, indice)=>{
    if(valor == maior){
        console.log(`${indice}`);
    }
});

console.log(`o menor valor digitado foi ${menor} nas posições`);
lista.forEach((valor, indice)=>{
    if(valor == menor){
        console.log(`${indice}`);
    }
});